_fr/tour/classes.md

```diff
@@ -7,6 +7,123 @@ num: 4
 
 language: fr
 
-next-page: traits
+next-page: default-parameter-values
 previous-page: unified-types
 ---
+
+Les classes en Scala sont des plans pour créer des objets. Elle peuvent contenir des méthodes,
```

Comment:
_plans_ feels weird, maybe _modèles_ or _gabarits_ ?

Reply:
I change _plans_ by _modèles_ which is better, ty.